<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>product</class>
 <widget class="QDialog" name="product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="corp_name">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Название фирмы</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="product_name">
     <property name="placeholderText">
      <string>Название продукта</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="shelf_life">
     <property name="placeholderText">
      <string>Срок годности</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="factory_price">
     <property name="placeholderText">
      <string>Цена на заводе</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
